### ISSUE TITLE
Valetudo stopped working after latest Update RE4?

### ISSUE THREAD

**Issue by nicedevil007** [rand256/valetudo#11]
Getting this: 

![image](https://user-images.githubusercontent.com/17103076/61524746-62332700-aa17-11e9-9acd-eb662e2a433d.png)

Did a stop of valetudo service and then overwrite the valetudo file inside of /usr/local/bin/. Then I doublechecked if the file got the execution mark and started the service. Nothing... still this error page now.

EDIT: tested with latest official PRE version RE3 => everything works as expected.
EDIT2: mhhh now I tested it with compiled version from your release. Everything is working. Maybe something changed for the docker container to compile it by myself?

**Comment by rand256** [comment]
Please see [deployment guide](https://github.com/rand256/valetudo/blob/testing/deployment/README.md).

With newer `pkg` node module it is a bit more complicated to build Valetudo yourself or use self-built version. You can however manually change pkg version back to 4.3.8 in `package.json` then everything most likely will work like before.

**Comment by nicedevil007** [comment]
Ok rand256, problem is fixed if I follow the "new" guide. Valetudo runs after service start with compiled version by myself.

But now:
Also I checked the 3 mentioned files... everything looks like the one from your deployment folder.
The map isn't working anymore :(

/etc/hosts

![image](https://user-images.githubusercontent.com/17103076/61581869-5b460a80-ab24-11e9-9c49-9a433defccda.png)

/etc/rc.local

![image](https://user-images.githubusercontent.com/17103076/61581875-7153cb00-ab24-11e9-9530-dcbebb2078c8.png)

/etc/init/valetudo.conf

![image](https://user-images.githubusercontent.com/17103076/61581883-8df00300-ab24-11e9-9e45-ec647a87844d.png)

Mhhh could this be a problem with your version and firmware 1720? Some people in https://www.roboter-forum.com/index.php?thread/24748-firmwareaktualisierungen/&postID=485213#post485213 noticed best behaviour of navigation for low profile furniture (I got a lowboard and the vacuum isn't cleaning under it... in some version before he did). Thats why I installed 1720 1-2 days before.
